### ISSUE TITLE
operations is not allowed in components section

### ISSUE THREAD

**Issue by KhudaDad414** [asyncapi/parser-js#858]
#### Describe the bug
Based on https://www.asyncapi.com/docs/reference/specification/v3.0.0-next-major-spec.12#componentsObject `components` section should allow `operations` in the components section.

#### How to Reproduce
parse an AsyncAPI file with:
```yaml
...
components:
  operations: {}
...
```
#### Expected behaviour
It should allow me to have operations in the components section.
I am not sure if it's a spec issue or a parser issue. 🤔

**Comment by derberg** [comment]
yup, operations should be possible under components -> https://github.com/asyncapi/spec/pull/806

imho it is not parser issue but json schema -> https://github.com/asyncapi/spec-json-schemas/blob/next-major-spec/definitions/3.0.0/components.json - operations are missing
